tests/sawtooth-pbft-test.dockerfile

```diff
@@ -35,12 +38,18 @@ RUN echo "deb [arch=amd64] http://repo.sawtooth.me/ubuntu/nightly xenial univers
     python3-requests \
     python3-nose2 \
     sawtooth-smallbank-workload \
-    sawtooth-smallbank-tp-go \
-    sawtooth-xo-tp-go \
+#    sawtooth-smallbank-tp-go \
```

Comment:
These and the below changes will go away as soon as we've published the latest Go SDK packages, and shouldn't be merged as-is.